### PR TITLE
main.less => main.css

### PR DESCRIPTION
**Pull request by amassarwi**
Closes https://github.com/decred/decrediton/issues/2608.

### CODE REVIEW

File: app/style/Layout.less [hunk 1]
```diff
@@ -1,7 +1,5 @@
 // Classes for laying out the pages
 
-@import (reference) "./main.less";
```
Comment (amassarwi) [inline app/style/Layout.less:3]: This PR should go in after #3278.

File: app/style/MyTickets.less [hunk 2]
```diff
@@ -1,5 +1,3 @@
-@import (reference) "./main.less";
```
Comment (amassarwi) [inline app/style/MyTickets.less:1]: This PR should go in after #3278.

**Review by matheusd — APPROVED** [review]

**Comment by alexlyp** [comment]
Needs a rebase after 3278

**Comment by amassarwi** [comment]
@alexlyp rebased

**Review by alexlyp — APPROVED** [review]

**Comment by amassarwi** [comment]
style lint need to be tweaked to work with .css instead of .less

**Review by alexlyp — APPROVED** [review]

**Pull request merged**